src/components/FormComponents/FormTextareaField/FormTextareaField.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import {
+  ErrorSpan,
+  Label,
+  TextArea,
+} from "components/FormComponents/FormTextareaField/FormTextareaField.styled";
+
+export interface FormTextareaFieldProps {
+  field: any;
+  form: { touched: any; errors: any };
+  placeholder: string;
+  step?: number;
+}
+
+const FormTextareaField: React.FC<FormTextareaFieldProps> = (props) => {
+  const { field, form, placeholder } = props;
```

Comment:
In case like this when you might or might not pass some params that are not a part of the library API (for instance `rows`, `cols`, `disabled`, `tabIndex` and so on) and there is no logic around them, you can always add `...rest` and then spread it on the component. This way you don't have to enumerate them all in your code, you just agree to pass whatever there is to the input.
```suggestion
  const { field, form, placeholder, ...rest } = props;
```